torchao/dtypes/aqt.py

```diff
@@ -480,6 +480,9 @@ def _apply_fn_to_data(self, fn):
         self.scale_and_zero = fn(self.scale_and_zero)
         return self
 
+    def __repr__(self):
+        return f"TensorCoreTiledAQTLayout(packed_weight={self.packed_weight}, scale_and_zero={self.scale_and_zero})"
```

Comment:
I think you can display the result of `get_plain()` here, packed_weight is not very useful
```
int_data, scale, zero_point = self.get_plain()
return f"TensorCoreTiledAQTLayout(int_data={int_data}, scale={scale}, zero_point={zero_point})"
```

Reply:
Ah ok sure!